bazel/BUILD

```diff
@@ -243,6 +243,22 @@ config_setting(
     values = {"define": "boringssl=disabled"},
 )
 
+config_setting(
+    name = "zlib_ng",
```

Comment:
It would be cool that instead of it being a bazel flag we could have zlib-ng be defined in config like the compression lib. But I understand that we need to do this at compile time, but not runtime. I guess you could compile both in and then chose at runtime, but that is not ideal either....

nothing actionable here, just a thought.